structs.go

```diff
@@ -153,6 +171,14 @@ func (s *Struct) Values() []interface{} {
 			}
 		}
 
+		if tagOpts.Has("string") {
+			s, ok := val.Interface().(fmt.Stringer)
+			if ok {
+				t = append(t, s.String())
+			}
+			continue
+		}
```

Comment:
Considering the possibility of an embedded struct being a Stringer, shouldn't the `string` tag check happen before the embedded struct check?